app/src/test/java/io/intrepid/contest/screens/contestcreation/editcategoriestocontest/EditCategoriesPresenterTest.java

```diff
@@ -2,78 +2,43 @@
 
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
 
 import io.intrepid.contest.models.Category;
-import io.intrepid.contest.testutils.BasePresenterTest;
+import io.intrepid.contest.testutils.TestPresenterConfiguration;
 
 import static junit.framework.Assert.assertTrue;
-import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.verify;
 
-public class EditCategoriesPresenterTest extends BasePresenterTest<EditCategoriesPresenter> {
-
+@RunWith(MockitoJUnitRunner.class)
+public class EditCategoriesPresenterTest {
```

Comment:
Same questions as last test class, why are you not extending `BasePresenterTest` anymore?